docs/src/api/class-electronapplication.md

```diff
@@ -0,0 +1,129 @@
+# class: ElectronApplication
+* langs: js
+
+Electron application representation. You can use [`method: Electron.launch`] to
+obtain the application instance. This this instance you can control main electron process
+as well as work with Electron windows:
+
+```js
+const { _electron: electron } = require('playwright');
+
+(async () => {
+  // Launch Electron app.
+  const electronApp = await electron.launch({ args: ['main.js'] });
+
+  // Evaluation expression in the Electron context.
+  const appPath = await electronApp.evaluate(async (electron) => {
+    // This runs in the main Electron process, |electron| parameter
+    // here is always the result of the require('electron') in the main
+    // app script.
+    return electron.getAppPath();
+  });
+
+  // Get the first window that the app opens, wait if necessary.
+  const window = await electronApp.firstWindow();
+  // Print the title.
+  console.log(await window.title());
+  // Capture a screenshot.
+  await window.screenshot({ path: 'intro.png' });
+  // Direct Electron console to Node terminal.
+  window.on('console', console.log);
+  // Click button.
+  await window.click('text=Click me');
+})();
+```
+
+## event: ElectronApplication.close
+
+This event is issued when the application closes.
+
+## event: ElectronApplication.window
+- type: <[Page]>
+
+This event is issued for every window that is created **and loaded** in Electron. It contains a [Page] that can
+be used for Playwright automation.
+
+## async method: ElectronApplication.close
+
+Closes Electron application.
+
+## method: ElectronApplication.context
+- type: <[BrowserContext]>
+
+This method returns browser context that can be used for setting up context-wide routing, etc.
+
+## async method: ElectronApplication.evaluate
+- returns: <[Serializable]>
+
+Returns the return value of [`param: expression`]
+
+If the function passed to the [`method: ElectronApplication.evaluate`] returns a [Promise], then
+[`method: ElectronApplication.evaluate`] would wait for the promise to resolve and return its value.
+
+If the function passed to the [`method: ElectronApplication.evaluate`] returns a non-[Serializable] value, then
+[`method: ElectronApplication.evaluate`] returns `undefined`. DevTools Protocol also supports transferring some additional
```

Comment:
Unrelated, but "DevTools Protocol"? Also, "bigint literals" are not supported, afaik.